Lucidtech/Las/Client.cs

```diff
@@ -826,10 +831,32 @@ public object CreateBatch(string? name = null, string? description = null)
             if (error != null) {
                 body.Add("error", error);
             }
+
+            if (startTime != null) {
+                body.Add("startTime", startTime);
+            }
+
             var request = ClientRestRequest(Method.PATCH, url, body);
             return ExecuteRequestResilient(RestSharpClient, request);
         }
 
+        /// <summary>Send heartbeat for a manual execution,
```

Comment:
"Send heartbeat for a manual execution to signal that we are still working on it. Must be done at minimum once every 60 seconds or the transition execution will time out"